gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/endpoint/login/LoginPostEndpoint.java

```diff
@@ -37,7 +37,10 @@
     public void handle(RoutingContext context) {
         final Session session = context.session();
         final MultiMap queryParams = RequestUtils.getCleanedQueryParams(context.request());
-        final String redirectUri = UriBuilderRequest.resolveProxyRequest(context.request(), context.get(CONTEXT_PATH) + "/oauth/authorize", queryParams, true);
+        final String redirectUri =
```

Comment:
Maybe we can move this URL builder step to a common method into the AbstractEndpoint class in order to avoid missing implementation if we have to change something? wdyt?